index.html

```diff
@@ -80,10 +82,10 @@ <h1 class="my-recent-works" id="portfolio">My Recent Works</h1>
       <!-- WORK 2 -->
       <div class="work-item">
         <div class="work-card">
-          <header class="work-card-title">
+          <h2 class="work-card-title">
             Multi-Post Stories<br>
             Gain+Glory
-          </header>
+          </h2>
           <ul class="work-card-tags nobull">
             <li class="work-card-tag">Ruby on rails</li>
             <li class="work-card-tag">css</li>
```

Comment:
- [ ] For the button for see project I think here you have low contrast errors so please make it normal you can use this extension to make it normal for all the button as you can see here  very low contrats between text and background image so kindly fix it, you can use this extension to check [wave](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

![image](https://user-images.githubusercontent.com/58553711/195073022-5b4a2e2d-526d-4635-a327-a1d8365c0add.png)

Reply:
Thank you! Done!